daffodil-lib/src/main/resources/org/apache/daffodil/xsd/XMLSchema_for_DFDL.xsd

```diff
@@ -40,10 +40,11 @@ this to validate when we load a DFDL Schema.
   -->
 
   <xsd:import namespace="http://www.ogf.org/dfdl/dfdl-1.0/" />
+    schemaLocation="DFDL_part3_model.xsd" />
```

Comment:
The line above has "/>" on it, so this line is broken/not well formed. Tests are not passing as a result.